lib/aws_account_utils/version.rb

```diff
@@ -1,3 +1,3 @@
 module AwsAccountUtils
-  VERSION = "0.1.2"
+  VERSION = "0.1.3"
```

Comment:
@keviny22  we do the version bump as a separate commit on master to make it easily identifiable.